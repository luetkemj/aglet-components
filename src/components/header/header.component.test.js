import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header.component';

test('Header renders correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
