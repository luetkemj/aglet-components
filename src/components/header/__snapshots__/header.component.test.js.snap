// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="masthead"
  >
    <div
      className="logo"
    >
      <img
        alt="logo"
        src="https://s3-us-west-2.amazonaws.com/agletio/images/nib.svg"
      />
    </div>
    <div
      className="nameplate"
    >
      <div
        className="name"
      >
        AGLET
      </div>
      <div
        className="description"
      >
        Tools for the discerning Game Master
      </div>
    </div>
  </div>
</div>
`;
